src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -906,6 +911,15 @@ public async Task HandleDidChangeTextDocumentNotification(ScriptFile[] changedFi
             }
         }
 
+        internal Task HandleRefreshTokenNotification(
+            TokenRefreshedParams refreshTokenParams,
```

Comment:
These names will also need to be updated - they should be `TokenRefreshed*`

Reply:
what do you mean by `TokenRefreshed*`? Isn't `TokenRefreshedParams` the right name?

Reply:
`HandleTokenRefreshedNotification`
`tokenRefreshedParams`